django/test/runner.py

```diff
@@ -405,15 +415,20 @@ def _init_worker(counter):
         counter.value += 1
         _worker_id = counter.value
 
+    if multiprocessing.get_start_method() == "spawn":
```

Comment:
We call `get_start_method()` multiple times (one of them in a loop below) in this function. Let's pull it out into a variable as we do elsewhere in this patch:

```python
    start_method = multiprocessing.get_start_method()
```